docs/reference/modules/node.asciidoc

```diff
@@ -1,32 +1,223 @@
-actual gather processing.
-
-This relieves the data nodes to do the heavy duty of indexing and
-searching, without needing to process HTTP requests (parsing), overload
-the network, or perform the gather processing.
+Any time that you start an instance of Elasticsearch, you are starting a
+**node** of Elasticsearch. A collection of connected nodes is what is
+called a <<modules-cluster,cluster>> of Elasticsearch. If you are running a
+single node of Elasticsearch, then you have a cluster of one node.
+
+Every node in the cluster can handle <<modules-http,HTTP>> and
+<<modules-transport,Transport>> traffic by default. The transport layer
+is used exclusively for communication from node-to-node; the HTTP layer
+is used only from external requests.
+
+Each node in the cluster serves one or more purposes, and there are
+four types of nodes:
+
+[cols="<,<,<,<",options="header",]
+|=======================================================================
+|Node Type |Setting |Default |Description
+|<<master,Master>> |`node.master` |`true` |A node that is eligible to control the cluster.
+|<<data,Data>> |`node.data` |`true` |A node that stores index data for search activity.
+|<<client,Client>> |`node.client` |`false` |A standalone node that routes requests, but
+does not hold data or handle master duties.
+|<<modules-tribe,Tribe>> |`tribe.*` |None |A special type of client node
+that allows you to connect to multiple clusters.
+|=======================================================================
+
+By default, if you do not touch any settings, then a node is both a
+master eligible node and a data node. This is extremely convenient for
+small clusters, such as if you create a test cluster locally on your own
+computer for development.
+
+NOTE: A master node is said to be _eligible_ because there is a
+<<modules-discovery-zen,master election process>> that determines which
+master node (if there is more than one) is elected.
+
+For larger clusters, it generally makes sense to have standalone data nodes
+and standalone master nodes. To do this, you must modify the configuration
+of the respective nodes.
+
+[float]
+[[master]]
+===== Master Eligible Node
+
+Creating a standalone master node has many benefits, but the most obvious
+is that it offloads work from another node. Master nodes control the
+cluster by determining where shards will exist, as well as determining
+what nodes are (and therefore are not) connected to the cluster.
+
+Master nodes _can_ receive search and indexing traffic from clients
+(e.g., you) and they will properly route the request, as will any other
+node, but you should never send these types of requests to a 
+standalone master node; standalone master nodes will inherently
+_never_ have data, which means that the request is an unnecessary
+burden. An easy way to think about it is: you are tasking the node to
+do work that is not part of its normal duties.
+
+To create a standalone master eligible node:
+
+```yaml
```

Comment:
format this as:

    [source,yaml]
    -------------------
    node.master: true
    node.data: false
    -------------------